frontend/src/components/pages/GlobalCodeReviewDashboard/FilterArea/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import Presentation from "./Presentation";
+
+export default function FilterArea({
+  filterIncludeTags,
+  filterExcludeTags,
+  filterIncludeFlavours,
+  filterExcludeFlavours,
+  handleChangeFlavourFilter,
+  handleChangeTagFilter,
+  allTagNames,
+  allFlavours,
+  allTeamNames,
+  filterIncludeAssigneeTeams,
+  handleChangeAssigneeTeamFilter,
+  cardNameSearchValue,
+  handleChangeCardNameSearchValue,
+}) {
+  const [flavourSearchTerm, setFlavourSearchTerm] = useState("");
```

Comment:
These search terms are in the wrong file. 

There should only be one call to useState to get this to work.

This code is not cohesive